add instructions to upgrade Control Center from 1.0 to 1.1 (#4043)
vaadin/docs: articles/control-center/getting-started/upgrade.adoc

---

> title: Upgrading from 1.0 to 1.1
> page-title: Upgrading from 1.0 to 1.1
> description: How to upgrade Control Center 1.0 to 1.1.
> meta-description: How to get upgrade Vaadin Control Center version 1.0 to 1.1.

Reviewer: I would avoid version numbers in the title and description, and keep this page version agnostic, so that we can reuse it for all future upgrade guides.

Also, `meta-description` is not needed if it’s the same as `description`.

```suggestion
title: Upgrading
page-title: Upgrading Vaadin Control Center
description: How to upgrade Vaadin Control Center from an older version to the latest version.
```

order: 110
---

= Upgrading from 1.0 to 1.1

> To upgrade Control Center from version 1.0.x to 1.1.x, you'll need to perform three tasks:

Reviewer: No need to explicitly imply patch/maintenance versions in my opinion. Omitting them implies all patch/maintenance versions.

```suggestion
To upgrade Control Center from version 1.0 to 1.1, you'll need to perform three tasks:
```

---
title: Upgrading from 1.0 to 1.1
page-title: Upgrading from 1.0 to 1.1
description: Learn how to upgrade Control Center 1.0 to 1.1
meta-description: Learn how to get upgrade Vaadin Control Center version 1.0 to 1.1.
order: 110
---

= Upgrading from 1.0 to 1.1

> To successfully upgrade Control Center from 1.0.X version to 1.1.X, you must perform three actions:

Reviewer 2: ```suggestion
To successfully upgrade Control Center from 1.0.x version to 1.1.x, you must perform three actions:
```

- Add a configuration property to the Control Center configmap.
- Create a secret that holds the email and password for the Control Center user that was created in the installation wizard.
- Set the docker image of the Control Center deployment to a 1.1 version.

NOTE: All of the example commands below assume that Control Center was installed in namespace named `control-center`.

== Add a New Necessary Configuration Property

> Control Center 1.1.0 uses a configuration property that did not exist in 1.0.

Reviewer 2: ```suggestion
Control Center 1.1 uses a configuration property that did not exist in 1.0.
```

If this property is missing, your deployment will not work after upgrading to 1.1.

> Edit the configmap for control center, adding the following under the `data` segment:

Reviewer 2: ```suggestion
Edit the configmap for Control Center, adding the following under the `data` segment:
```
